Library/Formula/namebench.rb

```diff
@@ -5,6 +5,31 @@ class Namebench < Formula
   url 'https://namebench.googlecode.com/files/namebench-1.3.1-source.tgz'
   sha1 '2e6ca5a4f20512cb967c5ac43b023cc38c271131'
 
+  head do
+    url 'http://namebench.googlecode.com/svn/trunk/'
+
+    patch :p0 do
+      url 'https://gist.github.com/catap/8230d76fba3c5d958428/raw/e57dcd26666d8a70dd482006e9c818c232cfa536/namebench-safari-8.patch'
+      sha1 '18e5066f187133636686feaefffefc22687a1197'
+    end
+
+    patch :p0 do
+      url 'https://gist.github.com/catap/8230d76fba3c5d958428/raw/af0a0750f023c041e01fe33f0febfb14e295045a/namebench-fix-second_level_domains.patch'
+      sha1 'a55c4b9580fd4b137e3d7f79eabaad1b349ac22a'
+    end
+
+    patch :p0 do
+      url 'https://gist.githubusercontent.com/catap/8230d76fba3c5d958428/raw/82c4a488f6c72d57f8427711c775bf12aee014c7/fix-config.patch'
+      sha1 '2a7eb58084c70598f02901e56b53269e874b9c49'
+    end
+
+    patch :p0 do
+      url 'https://gist.githubusercontent.com/catap/8230d76fba3c5d958428/raw/23373c6c0d5e733402faa2901663e2b1958bb922/fix-getip.patch'
+      sha1 'a47922d37afb7b6b8ee60b364b1b8cda11623399'
+    end
```

Comment:
I sent it.

But namebench trunk was changed at [May 2011 last time](https://code.google.com/p/namebench/source/list) and I don't know when patches will be accepted.